src/main/java/com/enderio/base/common/loot/CapacitorLootModifier.java

```diff
@@ -52,14 +53,30 @@ protected ObjectArrayList<ItemStack> doApply(ObjectArrayList<ItemStack> generate
         ItemStack capacitor = new ItemStack(EIOItems.LOOT_CAPACITOR.get());
         capacitor.getCapability(EIOCapabilities.CAPACITOR).ifPresent(cap -> {
             if (cap instanceof LootCapacitorData lootCap) {
-                lootCap.setBase(UniformGenerator.between(min, max).getFloat(context));
-                lootCap.addNewModifier(CapacitorUtil.getRandomModifier(context.getRandom()), UniformGenerator.between(0.0F, 4.5F).getFloat(context));
```

Comment:
Just noticed that these had different min/max. Might be worth either keeping it within the min/max or having another parameter in the loot modifier